api/swagger.yaml

```diff
@@ -2764,6 +2764,16 @@ definitions:
             type: "object"
             additionalProperties:
               type: "string"
+          MemorySwap:
+            description: "Total memory limit (memory + swap). Set as `-1` to enable unlimited swap."
+            type: "integer"
+            format: "int64"
+          MemorySwappiness:
+            description: "Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100."
```

Comment:
Would be good to describe which way it goes (i.e. if larger values means being more or less "swappy").